pages/resources/train-the-trainer.md

```diff
@@ -10,9 +10,24 @@ description: |
 objective: |
   The programme objective is to give instructors tools and tips for providing an enriching learning experience to trainees, irrespective of topic, and to include best-practice guidance on course and training material development. 
 
-contributors: [Piv Gopalasingam (EBI), Bruna Piereck (BE), Katarzyna Kamieniecka (UK), Krzysztof Poterlowicz (UK), Helena Schnitzer (DE), Lisanna Paladin (DE), Jessica Lindvall (SE), Piv Gopalasingam (EBI), Erin Calhoun (NO), Roland Krause (LU), Katharina Heil (Hub), Daniel Wibberg (DE), Renato Alves (DE)]
+contributors:
```

Comment:
I see there was some ELIXIR Node contribution captured previously, was that removed intentionally?

Reply:
We indeed have contributors as section for the resources. So, I'd keep it.

Reply:
The presence of the node information is what caused names be duplicated [here](https://elixir-europe-training.github.io/ELIXIR-Training-SPLASH/contributors) (see for instance my name and anyone else listed under contributors).

If the node information should be kept, then a different logic must be used in the contributors page to ensure names aren't repeated. At the time this PR was done, it seemed easier to remove the node information than to add it to the other source of contributors.

The contributors list wasn't removed but was instead converted to multi-line list and sorted alphabetically. This makes it more git-friendly and easier to maintain.

Reply:
Related, I just noticed that the PR will still not address all cases.
For instance, Krzystof is currently present three times:

```
KP - Krzys Poterlowicz
KP - Krzystof Poterlowicz
KP( - Krzysztof Poterlowicz (UK)
```

Reply:
We might need to centralize the information of each person so that they can appear in more than one resource. Need to check how this has been implemented elsewhere.

Reply:
> PR will still not address all cases

The way I understood the PR was that it's limited to TtT pages. Removing all duplicates is a great idea though.

> how this has been implemented elsewhere.

I guess duplicates are merged as long as everyone is listed in the exact same way across all pages.

As a different issue, perhaps one could implement a different way of enriching the personal info (picture, Node, orcid, github id, link to personal profile) without it having to be repeated in the contributors pages.